atlasdb-config/src/main/java/com/palantir/atlasdb/http/FailoverFeignTarget.java

```diff
@@ -112,13 +112,14 @@ private void checkAndHandleFailure(RetryableException ex) {
         boolean failedDueToNumSwitches = numSwitches.get() >= numServersToTryBeforeFailing;
 
         if (failedDueToFastFailover) {
-            log.warn("This connection has been instructed to fast failover for "
+            log.error("This connection has been instructed to fast failover for "
                     + TimeUnit.MILLISECONDS.toSeconds(fastFailoverTimeoutMillis)
                     + " seconds without establishing a successful connection."
                     + " The remote hosts have been in a fast failover state for too long.");
         } else if (failedDueToNumSwitches) {
-            log.warn("This connection has tried " + numServersToTryBeforeFailing
-                    + " hosts each " + failuresBeforeSwitching + " times and has failed out.", ex);
+            log.error("This connection has tried " + numServersToTryBeforeFailing
+                    + " hosts rolling across " + servers.size() + " servers, each "
```

Comment:
This wording is a bit confusing, though probably because the retrying logic is (imo) convoluted. My attempt to rephrase:

> We tried to connect a new host a total of {`numServersToTryBeforeFailing`} times, cycling through {`servers.size()`} distinct hosts. Each time, we made {`failuresBeforeSwitching`} attempts to connect to the host, for a total of {`failuresBeforeSwitching * numServersToTryBeforeFailing`} connection attempts. However, all these attempts failed out.